miqa/core/tasks.py

```diff
@@ -44,38 +45,39 @@ def evaluate_data(frame_ids, project_id):
         )
 
 
-def import_data(user_id, project_id):
-    project = Project.objects.get(id=project_id)
+def import_data(project_id: Optional[str]):
+    if not project_id:
+        project = None
+        import_path = GlobalSettings.load().import_path
+    else:
+        project = Project.objects.get(id=project_id)
+        import_path = project.import_path
 
-    if project.import_path.endswith('.csv'):
-        import_dict = import_dataframe_to_dict(pandas.read_csv(project.import_path))
-    elif project.import_path.endswith('.json'):
-        import_dict = json.load(open(project.import_path))
+    if import_path.endswith('.csv'):
+        import_dict = import_dataframe_to_dict(pandas.read_csv(import_path))
+    elif import_path.endswith('.json'):
+        import_dict = json.load(open(import_path))
```

Comment:
File handles should always be opened using a context manager, otherwise it's undefined behavior whether or not they'll be closed.

```suggestion
        with open(import_path) as import_fd:
            import_dict = json.load(import_fd)
```